libs/ccf-shared/src/lib/services/data-source/api-endpoint.service.ts

```diff
@@ -219,4 +203,28 @@ export class ApiEndpointDataSourceService implements DataSource {
       map((data) => (reviver ? reviver(data) : data)),
     );
   }
+
+  private checkForSources(): Observable<unknown> {
+    return combineLatest([this.globalConfig.getOption('dataSources'), this.globalConfig.getOption('filter')]).pipe(
+      debounceTime(1000),
```

Comment:
1000ms debounce seems a bit excessive. Let's work together to reduce/remove this